rpc/params/migration.go

```diff
@@ -121,6 +131,18 @@ type ModelArgs struct {
 	ModelTag string `json:"model-tag"`
 }
 
+// ActivateModelArgs holds args used to
+// activate a newly migrated model.
+type ActivateModelArgs struct {
+	ModelTag string `json:"model-tag"`
+
+	ControllerTag   string   `json:"controller-tag"`
+	ControllerAlias string   `json:"controller-alias,omitempty"`
+	SourceAPIAddrs  []string `json:"source-api-addrs"`
+	SourceCACert    string   `json:"source-ca-cert"`
+	CrossModelUUIDs []string `json:"cross-model-uuids"`
```

Comment:
Love comments for these public members too.

Reply:
added, was just following convention for other structs in this file.
i do think they are self explanatory though. most of the rpc params struct members are uncommented in general, and the comments which are there are extremely pithy